apps/etterlatte-beregning/src/main/kotlin/beregning/regler/barnepensjon/sats/BarnepensjonSats.kt

```diff
@@ -46,23 +46,55 @@ val antallSoeskenIKullet1967 = RegelMeta(
     regelReferanse = RegelReferanse(id = "BP-BEREGNING-1967-ANTALL-SOESKEN")
 ) benytter soeskenIKullet1967 med { soesken -> soesken.size }
 
+val harToAvdodeForeldre2024: Regel<BarnepensjonGrunnlag, Boolean> = finnFaktumIGrunnlag(
+    gjelderFra = BP_2024_DATO,
+    beskrivelse = "Finner om søker har to avdøde foreldre",
+    finnFaktum = BarnepensjonGrunnlag::harToAvdoedeForeldre,
```

Comment:
I denne funksjonen så mister vi strengt talt subsumsjonen ettersom metoden harToAvdoedeForeldre ikke inneholder grunnlaget og peker på en ekstern funksjon, men kun svaret på spørsmålet. FinnFaktumIGrunnlag bør nok egentlig bare returnere faktumet (listen med avdøde), og flytte den logikken til en ny regel, evt. gjøre det inne i barnepensjonSatsRegel2024.

Reply:
Fix in progress